test/reactionsystem.jl

```diff
@@ -9,11 +9,11 @@ KINETICMATH1 = SBML.MathIdent("k1")
 KINETICMATH2 = SBML.MathApply("*", SBML.Math[
 SBML.MathIdent("k1"), SBML.MathIdent("s2")])
 KINETICMATH3 = SBML.MathApply("-", SBML.Math[KINETICMATH2, KINETICMATH1])
-REACTION1 = SBML.Reaction(Dict("s1" => 1), Dict(), (NaN, ""), (NaN, ""), NaN,
+REACTION1 = SBML.Reaction(Dict(), Dict("s1" => 1), (NaN, ""), (NaN, ""), NaN,
```

Comment:
@paulflang Reaction1 is `1, 0->S1` right?

I think I just swapped the stoichs on accident

Reply:
> Reaction1 is 1, 0->S1 right?

yes.